league/dashboard/models.py

```diff
@@ -3,35 +3,97 @@
 
 from enum import Enum
 
-from league.database import Column, Model, SurrogatePK, db, reference_col
+from league.database import (Column, Model, SurrogatePK, association_proxy, db,
+                             reference_col, relationship)
 
 Color = Enum('Color', 'white black')
 
 
+class Player(SurrogatePK, Model):
```

Comment:
Correct. In this case, I manually execute the relevant query https://github.com/usgo/online-ratings/blob/master/web/app/views.py#L54
I should probably move this into a helper function so it's more obvious